README.md

```diff
@@ -315,3 +315,237 @@ If you are upgrading to a new (1.0.0 or newer) version of File, and you have pre
     cdvfile://localhost/persistent/path/to/file
 
 which can be used in place of the absolute file path in both `download()` and `upload()` methods.
+
+## Sample: Download and Upload Files
+
+Use the File-Transfer plugin to upload and download files. In these examples, we demonstrate several tasks like:
+
+* Downloading a binary file to the application cache
+* Uploading a file created in your application's root
+* Downloading the uploaded file
+
+## Download a Binary File to the application cache
+
+Use the File plugin with the File-Transfer plugin to provide a target for the files that you download (the target must be a FileEntry object). Before you download the file, create a DirectoryEntry object using `resolveLocalFileSystemURL`, passing in a Cordova file URL like `cordova.file.cacheDirectory`. Use the `getFile` method of DirectoryEntry to create the target file.
+
+```
+window.requestFileSystem(window.TEMPORARY, 5 * 1024 * 1024, function (fs) {
```

Comment:
No need to use `requestFileSystem` and `resolveLocalFileSystemURL` together to access the cache